manifests/deis-minio-service.yaml

```diff
@@ -7,8 +7,10 @@ metadata:
     heritage: deis
     release: 0.0.0
 spec:
+  type: NodePort
   ports:
     - port: 9000
+      nodePort: 32760
       targetPort: 9000
       name: s3
```

Comment:
Do we have any advantage adding node port. It makes sense only if you want to access minio with localhost:nodeport